services/app/src/quests/learning_to_adventure.xml

```diff
@@ -127,6 +133,7 @@
   <combat data-line="153">
     <e>Will o' the Wisp</e>
     <e>Magic Mushroom</e>
+    <e if="intimidated == 1">+2 to all attacks made during this combat.</e>
```

Comment:
I would explain to the user that this is because their intimidation succeeded

Reply:
I relocated this text to the combat tutorial screen and modified the text to mention that the bonus is because of the intimidation.